### ISSUE TITLE
Issue about the division of training set and test set of VQA data set in Table 4

### ISSUE THREAD

**Issue by 2133978609** [MIV-XJTU/FSDrive#87]
Thanks for yuor great work! I noticed in Table 4, you expand FSDrive to the VQA task validation on the DriveLM dataset. However, DriveLM does not officially divide the training set and test set. How do you divide the dataset in your work? I would be very grateful if you could provide the complete training set or test set of DriveLM.
